OloEngine/src/OloEngine/Renderer/AnimatedModel.cpp

```
@@ -1,16 +1,410 @@
+                    combined->GetIndices().Add(srcIndices[i] + baseVertex);
+                }
+
+                for (i32 i = 0; i < src->GetBoneInfluences().Num(); ++i)
+                {
+                    combined->GetBoneInfluences().Add(src->GetBoneInfluences()[i]);
+                }
+
+                for (i32 i = 0; i < src->GetShadowIndices().Num(); ++i)
+                {
+                    combined->GetShadowIndices().Add(src->GetShadowIndices()[i] + baseVertex);
+                }
```

Cppcheck (2.20.0)

[style] The function 'GetBoundingSphere' is never used.

(unusedFunction)